charts/cryostat/templates/reports_service.yaml

```diff
@@ -13,4 +13,5 @@ spec:
     - port: {{ .Values.reports.service.httpPort }}
```

Comment:
The report's Service should also have label `app.kubernetes.io/component: reports` for consistency?